patches/minecraft/net/minecraft/entity/player/EntityPlayer.java.patch

```diff
@@ -432,17 +432,27 @@
 +     * @param mod The mod trying to open a GUI
 +     * @param modGuiId GUI ID
 +     * @param world Current World
-+     * @param x Passed directly to IGuiHandler, data meaningless Typically world X position
-+     * @param y Passed directly to IGuiHandler, data meaningless Typically world Y position
-+     * @param z Passed directly to IGuiHandler, data meaningless Typically world Z position
++     * @deprecated, Use {@see #openGui(net.minecraftforge.gui.capability.IGuiProvider)} instead.
```

Comment:
you want `@link` here instead of `@see`

`@see` is used like `@param` or `@deprecated`.